docs/source/models/field_ionization.rst

```diff
@@ -121,6 +121,48 @@ One would expect much earlier ionization of Hydrogen due to lower ionization ene
 
```

Comment:
In your last sentence, you say that the plot explains why the BSI levels of Hydrogen and Carbon-II (in contrast to atomic Carbon?). But the plot alone misses an explanation. Could you add a sentence explaining why they are so close?

Reply:
Furthermore, the use of Carbon-II in both ADK cases but atomic-Carbon in one of the BSI plots from `models/field_ionization_comparison_c_ii_ionization.py` confuses me here.

Reply:
*offline discussion with @n01r:* 
This is not part of this pull request. This this should not be fixed here. I opened an issue  #3186 to address the problem.